solr-ingest/src/test/java/edu/unc/lib/dl/data/ingest/solr/filter/SetDescriptiveMetadataFilterTest.java

```diff
@@ -37,106 +41,247 @@
+	 */
+	@Test
+	public void testDateIssuedPreference() throws Exception {
+		SAXBuilder builder = new SAXBuilder();
+		Document modsDoc = builder.build(new FileInputStream(new File(
+				"src/test/resources/datastream/dateIssued.xml")));
+		when(dip.getMods()).thenReturn(modsDoc.detachRootElement());
+		
+		filter.filter(dip);
+		
+		verify(idb).setDateCreated(dateCaptor.capture());
+		Date dateIssued = dateCaptor.getValue();
+		SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM");
+		assertEquals("2006-05", dateFormat.format(dateIssued));
+	}
+	
+	/*
+	 * Covers case when there is only a dateCaptured field
+	 */
+	@Test
+	public void testDateCapturedPreference() throws Exception {
+		SAXBuilder builder = new SAXBuilder();
+		Document modsDoc = builder.build(new FileInputStream(new File(
+				"src/test/resources/datastream/dateCaptured.xml")));
+		when(dip.getMods()).thenReturn(modsDoc.detachRootElement());
+		
+		filter.filter(dip);
+		
+		verify(idb).setDateCreated(dateCaptor.capture());
+		Date dateCaptured = dateCaptor.getValue();
+		SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM");
+		assertEquals("2006-03", dateFormat.format(dateCaptured));
+	}
+	
+	@Test
+	public void testNamePartConcatenation() throws Exception {
+		SAXBuilder builder = new SAXBuilder();
+		Document modsDoc = builder.build(new FileInputStream(new File(
+				"src/test/resources/datastream/nameParts.xml")));
+		when(dip.getMods()).thenReturn(modsDoc.detachRootElement());
+		
+		filter.filter(dip);
+		
+		verify(idb).setCreator(listCaptor.capture());
+		assertTrue(listCaptor.getValue().contains("Repo, Boxy"));
+		verify(idb).setContributor(listCaptor.capture());
+		assertTrue(listCaptor.getValue().contains("Boxy"));
+		verify(idb).setCreatorSort(stringCaptor.capture());
+		assertEquals("Repo, Boxy", stringCaptor.getValue());
+	}
+	
+	@Test
+	public void testMultipleCreators() throws Exception {
+		SAXBuilder builder = new SAXBuilder();
+		Document modsDoc = builder.build(new FileInputStream(new File(
+				"src/test/resources/datastream/multipleCreators.xml")));
+		when(dip.getMods()).thenReturn(modsDoc.detachRootElement());
+		
+		filter.filter(dip);
+		
+		verify(idb).setCreator(listCaptor.capture());
+		assertTrue(listCaptor.getValue().contains("Test, Creator1"));
+		assertTrue(listCaptor.getValue().contains("Test, Creator2"));
+		verify(idb).setCreatorSort("Test, Creator1");
 
-		assertTrue(idb.getSubject().contains("Machine Learning"));
-		assertEquals(1, idb.getSubject().size());
+	}
+	
+	@Test
+	public void testAffiliationVocabTerm() throws Exception {
+		SAXBuilder builder = new SAXBuilder();
+		Document modsDoc = builder.build(new FileInputStream(new File(
+				"src/test/resources/datastream/inventoryMods.xml")));
+		when(dip.getMods()).thenReturn(modsDoc.detachRootElement());
+		Map<String, List<List<String>>> authTerms = new HashMap<String, List<List<String>>>();
```

Comment:
the HashMap declaration doesn't need to be parameterized, `HashMap<>` would be sufficient.  Same with `terms` and `depts`